docs/src/components/ColorContrastHelpers/ColorContrastHelpers.component.tsx

```diff
@@ -0,0 +1,32 @@
+// VENDOR
+import * as wcagContrast from 'wcag-contrast';
+// THEME
+import { colors } from '../../../../src/theme';
+
+
+export class ColorContrastHelpers {
+
+  public static getRoundedColorContrast(colorHexCode1, colorHexCode2): number {
+    let exactContrast = wcagContrast.hex(colorHexCode1, colorHexCode2);
+    return Math.round(exactContrast * 100) / 100;
+  }
+
+  public static getSuccessOrErrorColor(colorContrast): number {
+    if (colorContrast > 4.5) {
```

Comment:
In cases where you're using a single predicate, you can use a ternary to abbreviate this a bit.
```
return 1 > 2 ? true : false;
```

Reply:
✅ Updated